solr/modules/hdfs/src/test/org/apache/solr/hdfs/search/TestRecoveryHdfs.java

```diff
@@ -109,17 +109,13 @@ public static void afterClass() throws Exception {
     IOUtils.closeQuietly(fs);
     fs = null;
     try {
-      deleteCore();
```

Comment:
I wonder if it actually makes more sense to call the super afterclass method here instead - since that would clean stuff up in reverse order potentially. At least in the case of HDFS it makes sense to shutdown HDFS after shutting down solr I think...

Reply:
I did this in 74d2c561532cc6c335d3aa5cc8416511f0f51a48 and think its a lot cleaner.